streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractProcessorContext.java

```diff
@@ -198,4 +198,9 @@ public ThreadCache getCache() {
     public void initialized() {
         initialized = true;
     }
+
+    @Override
+    public void reinitialize() {
```

Comment:
need a better name for this method as `reinitialize` is suggesting that it is going to reinitialize something, but it isn't doing anything but resetting the initialized flag to false